Marlin/pins_PRINTRBOARD.h

```diff
@@ -74,11 +74,7 @@
 // Limit Switches
 //
 #define X_STOP_PIN         47   // E3
-#if ENABLED(SDSUPPORT)
-  #define Y_STOP_PIN       37   // E5 - Move Ystop to Estop socket
-#else
-  #define Y_STOP_PIN       20   // B0 SS - Ystop in Ystop socket
-#endif
+#define Y_STOP_PIN         20   // B0 SS
```

Comment:
Is this pin marked as "SS", as the comment says?

Reply:
B0 is the default SS pin for the Teensy++ 2.0:

![image](https://user-images.githubusercontent.com/1751183/39855041-847885e4-53e7-11e8-9b18-1991ac6e3af6.png)

I think that was the source of the original confusion. It's just that the Printrboard doesn't use the default SS pin for the SD card, it uses B6.

Reply:
Hmm, that's a good question. I think that PE2 and PE3 are the AVR register names, with corresponding pins on the IC, but those pins were probably just not given headers on the Teensy board. The Printrboard uses the same IC, hence it uses the same Arduino pin numbers as the Teensy (so I thought the diagram was useful), but since it's a standalone board, it can make use of all the registers it needs.

Reply:
Ah, makes sense. Of course, Printrboard is not Teensy, just "Teensy-like."